client/state/selectors/test/get-follow-count.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getFollowCount
+} from '../';
+
+describe( 'getFollowCount()', () => {
```

Comment:
these need to be updated because they no longer match the shape of state with the new reducer